### PR TITLE
ジャンル扱う変数名などををgenreに書き換えました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/constant/Genre.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public enum Genru {
 	private final int key;
 	private final String value;
 
-	private Genru(int key, String value) {
+	private Genre(int key, String value) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:22]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:22]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ctor def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/constant/Genre.java [hunk 2]
```diff
@@ -38,13 +38,13 @@ public String getValue() {
 	 * @param key　キー
 	 * @return　ジャンルを表す列挙型
 	 */
-	public static Genru of(int key) {
-		for (Genru genru : Genru.values()) {
-			if (genru.key == key) {
-				return genru;
+	public static Genre of(int key) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/constant/Genre.java [hunk 3]
```diff
@@ -38,13 +38,13 @@ public String getValue() {
...
+		for (Genre genre : Genre.values()) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/constant/Genre.java [hunk 4]
```diff
@@ -38,13 +38,13 @@ public String getValue() {
...
+			if (genre.key == key) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 24, expected level should be 6.

File: src/main/java/jp/co/example/ecommerce_a/constant/Genre.java [hunk 5]
```diff
@@ -38,13 +38,13 @@ public String getValue() {
...
+				return genre;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/jp/co/example/ecommerce_a/constant/Genre.java [hunk 6]
```diff
@@ -38,13 +38,13 @@ public String getValue() {
...
 			}
 		}
-		throw new IndexOutOfBoundsException("The value of genru doesn't exist");
+		throw new IndexOutOfBoundsException("The value of genre doesn't exist");
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/constant/Genre.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java [hunk 7]
```diff
@@ -49,14 +49,14 @@ public String showItemList(String name, Model model, String sortNum) {
 	 * 絞り込み検索したもちゃ一覧画面を出力します.
 	 * 
 	 * @param targetAge  対象年齢
-	 * @param genru      ジャンル
+	 * @param genre      ジャンル
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java:52]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java [hunk 8]
```diff
@@ -49,14 +49,14 @@ public String showItemList(String name, Model model, String sortNum) {
...
 	 * @param priceRange 価格帯
 	 * @param model      リクエストスコープ
 	 * @return おもちゃ一覧画面
 	 */
 	@PostMapping("/narrowingDown")
-	public String narrowingDown(Integer targetAge, Integer genru, Integer priceRange, Model model) {
-		List<Item> itemList = showItemListService.narrowingDown(targetAge, genru, priceRange);
+	public String narrowingDown(Integer targetAge, Integer genre, Integer priceRange, Model model) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java:58]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 104).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java:58]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java [hunk 9]
```diff
@@ -49,14 +49,14 @@ public String showItemList(String name, Model model, String sortNum) {
...
+		List<Item> itemList = showItemListService.narrowingDown(targetAge, genre, priceRange);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java:59]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java:59]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemListController.java:59]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 10]
```diff
@@ -59,31 +59,31 @@ public List<Item> findByName(String name) {
 	 * 対象年齢、ジャンル、価格帯でおもちゃを絞り込む.
 	 * 
 	 * @param targetAge　対象年齢
-	 * @param genru　ジャンル
+	 * @param genre　ジャンル
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused @param tag for 'genre　ジャンル'.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 11]
```diff
@@ -59,31 +59,31 @@ public List<Item> findByName(String name) {
...
 	 * @param priceRange　価格帯
 	 * @return　絞り込みされたおもちゃのリスト
 	 */
-	public List<Item> findByTargetAgeGenruAndPriceRange(Integer targetAge, Integer genru, Integer priceRange) {
+	public List<Item> findByTargetAgeGenreAndPriceRange(Integer targetAge, Integer genre, Integer priceRange) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 115).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 12]
```diff
@@ -59,31 +59,31 @@ public List<Item> findByName(String name) {
...
 		String sql = "Select id, name, description, price_m, price_l, image_path, deleted from items WHERE";
 		String sqlForTargetAge =" target_age IS NOT NULL";
-		String sqlForGenru =" AND genru IS NOT NULL";
+		String sqlForGenre =" AND genre IS NOT NULL";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '=' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3)

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 13]
```diff
@@ -59,31 +59,31 @@ public List<Item> findByName(String name) {
...
 		
 		MapSqlParameterSource addedParam = new MapSqlParameterSource();
 		if (targetAge!=null) {
 			sqlForTargetAge = " target_age=:targetAge";
 			addedParam.addValue("target_age", targetAge);
 		}
-		if (genru!=null) {
-			sqlForGenru = " AND genru=:genru";
-			addedParam.addValue("genru", genru);
+		if (genre!=null) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '!=' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3)

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '!=' is not preceded with whitespace.

**Review by SekiMoemi — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|78 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L78) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|78 col 25|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L78) 'if' child has incorrect indentation level 24, expected level should be 6.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|79 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L79) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|79 col 25|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L79) 'if' child has incorrect indentation level 24, expected level should be 6.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|86 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L86) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|86 col 17|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L86) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|42 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L42) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|46 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L46) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|46 col 9|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L46) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|48 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L48) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|48 col 17|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L48) 'if' has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|51|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L51) Line is longer than 100 characters (found 114).
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|51 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L51) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|51 col 25|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L51) 'else' child has incorrect indentation level 24, expected level should be 6.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|5 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L5) Extra separation in import group before 'jp.co.example.ecommerce_a.constant.Genre'
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|31 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L31) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|31 col 9|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L31) 'member def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|79 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L79) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|85|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L85) Line is longer than 100 characters (found 126).
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|85 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L85) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|94 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L94) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|94 col 17|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L94) 'ctor def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|156 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L156) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|156 col 9|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L156) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|157 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L157) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|157 col 17|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L157) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|160 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L160) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|160 col 9|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L160) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|161 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L161) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|161 col 17|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L161) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|191|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L191) Line is longer than 100 characters (found 119).
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|191 col 1|](http://github.com/ecsite-project/ec-202304a/blob/cce388d5612bb4cb0ab9f6dd4933ac6a7d001ad4/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L191) Line contains a tab character.
</details>

**Review by yamada-kensyu — APPROVED** [review]
修正ありがとうございます！
承認します

**Pull request merged**